### ISSUE TITLE
🌐 Website Issue | Forum displays blank page in Firefox-based browsers, intermittent

### ISSUE THREAD

**Issue by lunkhub** [privacytools/privacytools.io#2399]
## Description
forum.privacytools.io intermittently displays as blank page in Tor browser on phone or desktop, or Mull with orbot on phone. Page title and onion option are shown, on desktop; see screenshot. View source displays long, long source. Privacy Browser (webview-based) with orbot on phone displays OK. It is not a browser issue because all other websites work OK or display "blocked/robot/we-hate-tor" pages, as usual. This started in the last week or so.

## Screenshots

![Screenshot_2021-08-01-1x](https://user-images.githubusercontent.com/66975721/127771844-4c975bbc-c4ac-4cf0-a53d-fb56168a596e.png)
![Screenshot_2021-08-01-2x](https://user-images.githubusercontent.com/66975721/127771847-65ac5dbd-ba95-47c5-a87e-5eed21231937.png)

**Comment by dngray** [comment]
Hmm, I wonder if this happened when I updated it.

**Comment by Atlantistux** [comment]
I have the same with Firefox ESR 78.13 and Vanadium 92 (Chromium).

**Comment by HatanM** [comment]
https://forum.privacytools.io/ page is blank in ANY browser since about a months ...

**Comment by Dyrimon** [comment]
Indeed, the whole forum is just black screen. Although it says "Operational" fr forum in status.privacytools.io
